modules/github_integration/app/models/github_pull_request.rb

```diff
@@ -68,6 +74,10 @@ def latest_check_runs
                      .order(app_id: :asc, name: :asc, started_at: :desc)
   end
 
+  def partial?
+    [body, comments_count, review_comments_count, additions_count, deletions_count, changed_files_count].all?(&:nil?)
+  end
```

Comment:
From what I see the `partial?` method is used only in the `validate_presence_of` call above. I would suggest to just remove the presence validation for these fields altogether and also remove the `partial?` method. Then we can get rid of this unholy concept of PRs being "partial".

Reply:
I'd leave it in for now as we still don't have all the information at hand on a pr created via a comment e.g the runs and changes, compared to a full pr. So it might be beneficial to have that distinction still. I'd remove it once it gets in the way of things, though.